include/seqan3/alignment/matrix/detail/aligned_sequence_builder.hpp

```diff
@@ -174,8 +176,27 @@ class aligned_sequence_builder
 
         matrix_coordinate path_begin = trace_it.coordinate();
 
-        fst_aligned_t fst_aligned{fst_rng | views::slice(path_begin.col, path_end.col)};
-        sec_aligned_t sec_aligned{sec_rng | views::slice(path_begin.row, path_end.row)};
```

Comment:
Oh boy, I see the problem I have missed before. A vector cannot be created from a range. And the gap decorator cannot  be created from a pair of iterators 🤦‍♂ . Well, I need to think about this. I had introduced a type trait class in order to allow the usage of it in a more generic context. This however, defeats the purpose of it. I will rethink the design. But it feels more like a policy based design then.

Reply:
There is the `assign_unaligned` function that lets you assign a range to a vector or gap_decorator with a common interface